### DISCUSSION TITLE
Light theme

### DISCUSSION THREAD

**Discussion by Ansh4011** [aleksey-hoffman/sigma-file-manager#74] [Show and tell]
Hi @aleksey-hoffman! I really like the SIGMA file manager, it is one of it's own kind, and SALUTE to your hard efforts! I am kinda in love with the file manager! I'll be testing the software and give the reports to you here on github! Also, I have designed how the light mode of Sigma File Manager maybe would look like! Check the reference pic! What do you opine for light mode? Does it stand a chance? And is it worth giving it a try? I have lots of questions....
![Light Mode](https://user-images.githubusercontent.com/84930340/120000086-0f056480-bff0-11eb-8385-a193b8320595.jpg)

![Dark Mode](https://user-images.githubusercontent.com/84930340/120000074-0d3ba100-bff0-11eb-9baf-9dfd62b61f91.png)

**Comment by Ansh4011** [comment]
I'll be eagerly waiting for your reply @aleksey-hoffman! 😀

**Comment by aleksey-hoffman** [comment]
Thank you for the feedback mate. I'm glad you're enjoying the app :)

I like the colors in your design. 

You would be surprised, but the app had a light theme from the very beginning. I removed the light theme option from settings a few months ago because it wasn't ready for the release, I didn't have time to finish it.

I'm not sure if many people would want to see a light theme, but I will finish the light theme anyway.
I just need to fix other more important problems first.

**Reply by Ansh4011** [reply]
> Thank you for the feedback mate. I'm glad you're enjoying the app :)
> 
> I like the colors in your design.
> 
> You would be surprised, but the app had a light theme from the very beginning. I removed the light theme option from settings a few months ago because it wasn't ready for the release, I didn't have time to finish it.
> 
> I'm not sure if many people would want to see a light theme, but I will finish the light theme anyway.
> I just need to fix other more important problems first.

Totally Agreed! Thanks for the feedback😀! Take your time for the Light Theme! Black theme looks cool too😎

**Comment by aleksey-hoffman** [comment]
As for the custom icons, I can add support for it later as well

**Reply by Ansh4011** [reply]
> As for the custom icons, I can add support for it later as well

Yeah sure! Do let me if I could help in designing the icons! In fact, currently I have my vacations so maybe I could get it done pretty quickly!😀

**Comment by KniggeMS** [comment]
@Ansh4011 Your light theme looks great! @aleksey-hoffman Would it be difficult to implement as an extension in Theme Manager?
